### ISSUE TITLE
Provide support for pseudo elements like :before and :after in specs

### ISSUE THREAD

**Issue by ishubin** [galenframework/galen#327]
At the moment it is not possible to test :before and :after elements in spec.

**Comment by renet** [comment]
+1 from me

How come this issue has been removed from the milestones? It's common to use those for CSS...

**Comment by ishubin** [comment]
@renet It was removed because I couldn't come up with a reliable solution for it and at some point I just gave up. Hope to find out something working in future

**Comment by Prawin10690** [comment]
@ishubin still galen cant identify pseudo elements?

**Comment by ishubin** [comment]
@Prawin10690 yes, this issue is still open. still haven't found a solution to it
